main.rb

```diff
@@ -0,0 +1,74 @@
+require './app'
+
+class LibraryApp
+  def display_options
+    puts 'Options:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List rentals for a person'
+    puts '7. Quit'
+  end
+
+  def list_rentals(people)
+    print 'Enter person id: '
+    person_id = gets.chomp.to_i
+    list_rentals_for_person(person_id, people)
+  end
+
+  def handle_list_books(books)
+    list_all_books(books)
+  end
+
+  def handle_list_people(people)
+    list_all_people(people)
+  end
+
+  def handle_create_person(people)
+    create_person(people)
+  end
+
+  def handle_create_book(books)
+    create_book(books)
+  end
+
+  def handle_create_rental(books, people)
+    create_rental(books, people)
+  end
+
+  def handle_list_rentals(people)
+    list_rentals(people)
+  end
```

Comment:
[OPTIONAL]
 - I think there is no need to make another method and then call another method. you can directly call `create_book` or `create_person` in `handle_choice` method.